pipeline/07-export.R

```diff
@@ -271,40 +271,71 @@ assessment_card <- dbGetQuery(
 #- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 message("Preparing data for Desk Review export")
 
-# Get the number of apartments for class 211 and 212 properties. Since
-# char_apts is recorded on the card level, we need to aggregate them to
-# the PIN level
-num_apts_by_pin <- assessment_card %>%
-  filter(char_class == "211" | char_class == "212") %>%
-  select(meta_pin, char_apts) %>%
-  # Convert the long format for char_apts to a shorter one that's easier
+format_char_apts <- function(char_apts) {
+  # Convert the long format for char_apts to a shorter format that's easier
   # to scan in a spreadsheet
-  mutate(
-    char_apts = case_when(
-      char_apts == "NONE" | is.na(char_apts) ~ NA_character_,
-      char_apts == "TWO" ~ "2",
-      char_apts == "THREE" ~ "3",
-      char_apts == "FOUR" ~ "4",
-      char_apts == "FIVE" ~ "5",
-      char_apts == "FIX" ~ "6",
+  return(
+    case_when(
+      is.na(char_apts) | tolower(char_apts) == "none" ~ NA_character_,
```

Comment:
Since `char_apts` is formatted in title case in the training data (e.g. `None`) but all caps in the assessment data (e.g. `NONE`), `tolower()` allows us to align the two formats when transforming the output.